code/modules/organs/internal/brain.dm

```diff
@@ -198,6 +200,8 @@
 					damprob = owner.chem_effects[CE_STABLE] ? 30 : 60
 					if(!past_damage_threshold(2) && prob(damprob))
 						take_internal_damage(1)
+					if(can_heal)
```

Comment:
Making sure that brain atrophy is better treatable, because as I said, brain damage is accrued way too fast, and if a person wears a rig, there's just no way they can be recovered. And everyone wears rigs.

Reply:
the brain is supposed to accrue damage relatively fast so that people aren't lying in crit forever. Though we should have the means to save people if they can be recovered

Regarding alkysine, it would be best to rework its code to not be dependant on this at all, just have it fetch the brain from affect_blood

revert these brain healing changes. But if the goal is to make critical injuries more treatable, consider buffing inaprovaline instead. IIRC it's handled in another proc, but it contributes a large quantity to brain oxygenation

Reply:
I'll have to do some research, but I'll get back to you on this.